Automattic/jetpack
# Blocks: update recipe category block

Fixes #28370

## Proposed changes:

The Recipe block, currently in Beta, uses the `jetpack` category, which doesn't exist. It generates notices in environments where you use the Beta bundle.

I've updated the block and its children blocks to the "widgets" category, to get rid of the notices.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Launch a site with Beta blocks enabled
* Go to Posts > Add New
* Open JavaScript console; notice new issues.
